CHANGELOG.md

```diff
@@ -190,12 +192,14 @@ This version of the dashboard requires Cumulus API `v8.1.0`
 - **CUMULUS-2449**
   - Fixes issue where collections with forward slash in version name were not displayed
 
-_ **CUMULUS-2425**
-  - Fixes graph display for failed execution steps
+\_ **CUMULUS-2425**
```

Comment:
The `\` looks like a typo

Reply:
Ha ha. That's weird. Lint formatting having too much fun.